features/address-bar-tld-nav-or-search.json

```diff
@@ -0,0 +1,7 @@
+{
+    "_meta": {
+        "description": "Use TLD based domain checks in the address bar for search or nav logic on Windows Browser"
```

Comment:
Should we remove mention of `Windows Browser` now that the feature is not name to be windows specific?